### PR TITLE
4 packages from c-cube/calculon at 0.5

### PR DESCRIPTION
**Pull request by c-cube**
This pull-request concerns:
-`calculon.0.5`: Library for writing IRC bots in OCaml and a collection of plugins
-`calculon-redis.0.5`: A redis plugin for Calculon
-`calculon-redis-lib.0.5`: A library to interact with Calculon via Redis
-`calculon-web.0.5`: A collection of web plugins for Calculon

---
* Homepage: https://github.com/c-cube/calculon
* Source repo: git+https://github.com/c-cube/calculon.git
* Bug tracker: https://github.com/c-cube/calculon/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by c-cube** [comment]
ugh, as usual, mdx :(
tests would have been ignored with the .travis.yml, cf https://travis-ci.org/c-cube/calculon/builds/609685637

File: packages/calculon-redis/calculon-redis.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  ["dune" "build" "@doc" "-p" name] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" { >= "1.1" }
+  "calculon" { = version }
+  "calculon-redis-lib" { = version }
+  "redis" { >= "0.3" }
+  "redis-lwt"
+]
```
Comment (kit-ty-kate) [inline packages/calculon-redis/calculon-redis.0.5/opam]:
```suggestion
  "mdx" {with-test}
]
```
```
#=== ERROR while compiling calculon-redis.0.5 =================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/c-cube/calculon/archive/v0.5.tar.gz)
# path                 ~/.opam/4.07/.opam-switch/build/calculon-redis.0.5
# command              ~/.opam/4.07/bin/dune runtest -p calculon-redis -j 2
# exit-code            1
# env-file             ~/.opam/log/calculon-redis-29-ffb3fd.env
# output-file          ~/.opam/log/calculon-redis-29-ffb3fd.out
### output ###
# File "dune", line 10, characters 15-24:
# 10 |           (run ocaml-mdx test %{deps})
#                     ^^^^^^^^^
# Error: Program ocaml-mdx not found in the tree or in PATH
#  (context: default)
```

File: packages/calculon-redis-lib/calculon-redis-lib.0.5/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" { >= "1.1" }
+  "redis" { >= "0.3" }
+  "calculon" { = version }
+  "redis-lwt"
+  "atdgen"
+  "yojson"
+]
```
Comment (kit-ty-kate) [inline packages/calculon-redis-lib/calculon-redis-lib.0.5/opam]:
```suggestion
  "mdx" {with-test}
]
```
```
#=== ERROR while compiling calculon-redis-lib.0.5 =============================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.03.0 | pinned(https://github.com/c-cube/calculon/archive/v0.5.tar.gz)
# path                 ~/.opam/4.03/.opam-switch/build/calculon-redis-lib.0.5
# command              ~/.opam/4.03/bin/dune runtest -p calculon-redis-lib -j 2
# exit-code            1
# env-file             ~/.opam/log/calculon-redis-lib-29-ffb3fd.env
# output-file          ~/.opam/log/calculon-redis-lib-29-ffb3fd.out
### output ###
# File "dune", line 10, characters 15-24:
# 10 |           (run ocaml-mdx test %{deps})
#                     ^^^^^^^^^
# Error: Program ocaml-mdx not found in the tree or in PATH
#  (context: default)
```

**Comment by kit-ty-kate** [comment]
Could you also add strict constraints on mdx? The author of the package says it shouldn't use in tests without strong constraints. See https://github.com/ocaml/opam-repository/pull/14970#issuecomment-537451688

**Comment by c-cube** [comment]
Not sure what I should do, except for just dropping mdx? :/

**Comment by kit-ty-kate** [comment]
Thanks! Sorry for the delay

**Comment by c-cube** [comment]
Thanks @kit-ty-kate ! No worries, I saw that opam-ci's queue was delayed by some massive rebuilds :slightly_smiling_face:

**Pull request merged**